.github/workflows/dev_pr/labeler.yml

```
@@ -15,53 +15,53 @@
 # specific language governing permissions and limitations
 # under the License.
 
-"lang-c++":
+"Component: c++":
   - cpp/**/*
 
-lang-c-glib:
+"Component: c-glib":
   - c_glib/**/*
 
-lang-csharp:
+"Component: csharp":
   - csharp/**/*
 
-lang-go:
+"Component: go":
   - go/**/*
 
-lang-java:
+"Component: java":
   - java/**/*
 
-lang-js:
+"Component: js":
```

```suggestion
"Component: JavaScript":
```

On the short term, we should probably match the labels exactly as the components on JIRA (we can rename them after the issues migration)? So for example "js" is "JavaScript"

---

Thanks! I have updated all the labels to match the existing ones on JIRA.